### DISCUSSION TITLE
Avalanche chain removed in wagmi@0.3.0

### DISCUSSION THREAD

**Discussion by real-lazy-coder** [wevm/wagmi#385] [Q&A] [answered]
Why was avalanche removed from wagmi as a default?

```ts
import { chain } from "wagmi";

const defaultChain = chain.avalanche;

```
☝️ code no longer works in wagmi@0.3.0

**Comment by tmm** [comment, marked as answer]
Yeah! Need to figure out a better process for adding chains to the core. In the meantime, you can create your own chains by conforming to the [`Chain` type](https://github.com/tmm/wagmi/blob/main/packages/core/src/types/index.ts#L8-L23=). Related discussion https://github.com/tmm/wagmi/discussions/370#discussioncomment-2650012

**Reply by real-lazy-coder** [reply]
Yeah...  That's what I did.  Just wasn't sure if there was an issue with avalanche as to why it was removed.

Thanks,

**Reply by tmm** [reply]
Wanted to pare back to the original base while coming up with a process for adding new chains since there are [a lot of them](https://chainlist.org)!

**Reply by hotarumoon** [reply]
I have added avalanche with a Chain type as well but in that case I cannot use the useBalance hook it gives this error: POST https://rpc.walletconnect.com/v1/?chainId=eip155:43113&projectId=9b8b1dd22536a951907958fe4db2cf03 400 (Bad Request)  with this response: {"status":"FAILED","reasons":[{"field":"chainId","description":"We don't support the chainId you provided: eip155:43113"}]}

**Comment by jxom** [comment]
With the introduction of `0.9.x`, wagmi now comes with the `avalanche` & `avalancheFuji` chains built-in (re-exported from our [`@wagmi/chains` reference repository](https://github.com/wagmi-dev/references/tree/main/packages/chains)). Read more [here](https://wagmi.sh/react/chains).
